cfg/TebLocalPlannerReconfigure.cfg

```
@@ -141,6 +141,10 @@ grp_robot_omni = grp_robot.add_group("Omnidirectional", type="hide")
 
 grp_robot_omni.add("max_vel_y", double_t, 0, 
   "Maximum strafing velocity of the robot (should be zero for non-holonomic robots!)",
+  0.0, 0.0, 100)
+
+grp_robot_omni.add("max_vel_linear", double_t, 0, 
+  "Maximum linear velocity of the robot. Ignore this parameter for non-holonomic robots (max_vel_linear == max_vel_x).",
   0.0, 0.0, 100) 
```

```suggestion
  0.4, 0.01, 100) 
```

S: Use the same values as `max_vel_x`.